module-system/module-implementations/sov-bank/src/call.rs

```diff
@@ -8,10 +8,15 @@ use crate::{Amount, Bank, Coins, Token};
 #[cfg_attr(
     feature = "native",
     derive(serde::Serialize),
-    derive(serde::Deserialize)
+    derive(serde::Deserialize),
+    derive(schemars::JsonSchema),
+    schemars(bound = "C::Address: ::schemars::JsonSchema", rename = "CallMessage")
```

Comment:
(Question) What name does `schemars` generate without the `rename = CallMessage` annotation?

Reply:
One more potential option: we could wrap `derive(JsonSchema)` in our own macro which (for now), just adds the `JsonSchema` annotation with `rename`. That way, we can also change out our schema generator in future without breaking consumers

Reply:
That's an attractive option, the problem I see with that is it's quite leaky by necessity. Anytime one needs to add a `#[schemars(...)]` attribute on a field or use some of the more advanced features –we didn't need to do it for `sov-bank`, luckily– they will have to go through our proc macro and still inevitably take into consideration Schemars implementation details.

I say we reevaluate after all built-in modules have a schema. If we see there's no need for meddling with Schemars internals, we can then assume most users won't have to care about it and we can wrap it. Otherwise, we evaluate one of the other options. WDYT?

Reply:
That's a good point. Sounds fine to me!